docs/admin/accessing-the-api.md

```diff
@@ -52,8 +52,10 @@ On GCE, Client Certificates, Password, Plain Tokens, and JWT Tokens are all enab
 If the request cannot be authenticated, it is rejected with HTTP status code 401.
 Otherwise, the user is authenticated as a specific `username`, and the user name
 is available to subsequent steps to use in their decisions.  Some authenticators
-may also provide the group memberships of the user, while other authenticators
-do not (and expect the authorizer to determine these).
+also provide the group memberships of the user, while other authenticators
+do not.  An authorizer may determine as well as use groups (or any other
```

Comment:
It's unclear what you mean by "determine". I think you're saying the authorizer could use groups from some other source, or could itself be a source of groups, or neither. Defining "determine" might help.

also please squash when you clarify, thanks.